vagrant/provisioning/roles/alfresco/tasks/main.yml

```diff
@@ -474,6 +474,7 @@
   uri:
     url: https://localhost:7070/share/page/
     validate_certs: false
+    follow_redirects: all
```

Comment:
is the installer working now with this change?

Reply:
Yes, we tried the change locally on the arkcase-ce aws instance that we are preparing for the MarketPlace